evalml/model_understanding/permutation_importance.py

```diff
@@ -325,5 +325,6 @@ def _fast_scorer(pipeline, features, X, y, objective):
         preds = pipeline.estimator.predict_proba(features)
     else:
         preds = pipeline.estimator.predict(features)
+    preds = pipeline.inverse_transform(preds)
```

Comment:
If I'm following correctly, we only need to change the fast_scorer because in the slow_scorer we call score --> predict --> predict_in_sample which does inverse_transform, correct?

Reply:
Yea - this is why the slow score is the correct one. In the "slow" method we basically treat the whole pipeline as an estimator. So we call `pipeline.predict`, which does `pipeline.inverse_transform`. In the fast method, we optimize by only doing the feature engineering once and get the predictions by doing `estimator.predict`. Since the estimator has no idea it's part of a pipeline, we need to remember to call `inverse_transform`.